.clang-format

```diff
@@ -79,6 +79,8 @@ ForEachMacros:
   - 'HTTP_SERVER_CONTENT_TYPE_FOREACH'
   - 'HTTP_SERVICE_FOREACH'
   - 'HTTP_SERVICE_FOREACH_RESOURCE'
+  - 'i3c_bus_for_each_i3cdev'
+  - 'i3c_bus_for_each_i2cdev'
```

Comment:
I'm not quite sure if these "have to" be capitals... but if so it's easy to change